features/addons-ext/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.distro</groupId>
+    <artifactId>pom-features</artifactId>
+    <version>2.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>addons-ext</artifactId>
+  <packaging>feature</packaging>
+
+  <name>openHAB Add-ons External</name>
```

Comment:
"External" might not be the best name, it sounds as if they do not belong to openHAB.

I also do not understand the reason why we need a different feature for them - from an user point of view, they should just appear in the very same way as any other add-ons.

Reply:
See below, the intention as I created it has been to have on aggregator feature without feature definition itself.

Reply:
As we dropped the aggregate "addons" feature, we could simply name this one here "addons" now.